config.json

```diff
@@ -107,6 +107,16 @@
       ]
     },
     {
+      "uuid": "af5ccf14-eff2-4dc6-b1db-e209cddca62a",
+      "slug": "rotational-cipher",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+
```

Comment:
It would be good to add some topics, see 'collatz-conjecture' for an example and https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt for a list of topics.